backend/src/DataTypes.d.ts

```diff
@@ -1,23 +1,41 @@
 import { firestore } from 'firebase-admin';
 
-export type Role = 'lead' | 'admin' | 'tpm' | 'pm' | 'developer' | 'designer';
+export type Role =
+  | 'lead'
+  | 'admin'
+  | 'tpm'
+  | 'pm'
+  | 'developer'
+  | 'designer'
+  | 'business';
+
+export type roleDescription =
```

Comment:
Can we standardize to use `PascalCase` here?

Reply:
done!